File: api/ServerManager.Rest/Dto/Template.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServerManager.Rest.Dto
{
    public class Template
    {
        public int TemplateId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Version { get; set; }
        public string DownloadLink { get; set; }
        public ServerPropertyList Properties { get; set; }
    }
}

File: ui/management-console/src/models/Template.ts
import { ServerProperty } from "./ServerProperty";

export class Template {
	public constructor() {
		this.templateId = 0;
		this.name = "";
		this.description = "";
		this.version = "";
		this.downloadLink = "";
		this.properties = [];
	}

	templateId: number;
	name: string;
	description: string;
	version: string;
	downloadLink: string;
	properties: ServerProperty[];
}
